core-bundle/assets/controllers/clipboard-controller.js

```diff
@@ -1,25 +1,43 @@
 import { Controller } from '@hotwired/stimulus';
 
-export default class extends Controller {
+export default class ClipboardController extends Controller {
+    static instances = [];
+    static classes = ['written'];
     static values = {
         content: String
     }
 
-    write () {
-        if (navigator.clipboard && navigator.clipboard.writeText) {
-            navigator.clipboard.writeText(this.contentValue).catch(this.clipboardFallback.bind(this));
-        } else {
-            this.clipboardFallback();
+    connect() {
+        if (!ClipboardController.instances.contains(this)) {
+            ClipboardController.instances.push(this);
         }
     }
 
-    clipboardFallback  () {
-        const input = document.createElement('input');
-        input.value = this.contentValue;
-        document.body.appendChild(input);
-        input.select();
-        input.setSelectionRange(0, 99999);
-        document.execCommand('copy');
-        document.body.removeChild(input);
```

Comment:
@ausi intentionally added clipboard fallback, why was this removed?

Reply:
Judging from the browserslist we do not support such old browsers in Contao 5, do we?